[WIP] Add Farsi translation

huey735 opened the pull request:
For some reason "How does the Bisq DAO work?" under bisq.network/fa/faq won't expand to show the text. @wiz @m52go @erciccione

huey735 committed:
Add outdated_translation variable

On @m52go's advice I changed the construction of the outdated
translation alert tool to be spefic to the page instead of
the language. I added a "outdated_translation" variable to
the front matter of the relevant translated pages. They'll
have the values 'true' or 'false' according to the state.

erciccione committed:
Refactored "Statistics" page to make it more i18n friendly

Data and dates needed to be manually updated for all languages every month. After this change will be enough to only change the value in includes/statistics_data.html, the other languages will pull the data from there. I also added generic time references ('Through last month' instead of 'July'), to remove the need to translate every time the name of the past month.

huey735 committed:
Add Farsi translations

huey735 committed:
Add Farsi translations

huey735 committed:
Deplace outdated_translation variable

Move outdated_translation tool from _includes/main_nav_tr.html
to _layouts/page.html.

Commit:
Separate total volume by fiat and altcoin

huey735 committed:
Make warning text more visible

Changed the text color to black and background-color to orange.
And on hover the text-decoration is underline and the color is
blue.

huey735 committed:
Fix spacing and markdown formatting

huey735 committed:
Fix spacing and markdown formatting

Commit:
Add fiat-altcoin change to all stats pages

Also use variable for chart caption text.

huey735 committed:
Fix typo

huey735 committed:
Fix typo

huey735 committed:
Merge branch 'master' into addFarsi

huey735 committed:
Make warning text fit the website theme

huey735 committed:
Remove unnecessary div

Commit:
Merge pull request #226 from m52go/separate-altcoin-fiat

Separate volume by fiat and altcoin

Commit:
Merge pull request #221 from erciccione/refactor-statistics

Refactored "Statistics" page to make it more i18n friendly

Commit:
Merge pull request #220 from huey735/addOutdatedTranslationVariable

Add outdated_translation variable

Commit:
Update version for current software stats

Commit:
Set all stats `outdated_translation` to false

Commit:
Add `outdated_translation` variable to fr pages

Commit:
Move altcoin chart caption below chart

Commit:
Merge pull request #228 from m52go/small-corrections

Update software stats version and outdated_translation vars

Commit:
Add front-matter for blog post site header

To apply fix from #212 to this post.

wiz committed:
Add Japanese localizations

wiz committed:
Display localized app screenshots, use english placeholders for now

wiz committed:
Add localized app screenshots for Japanese

wiz committed:
Add missing "fr" and "fr-FR" locales in languages.yml

Commit:
Merge pull request #231 from wiz/fix-french-redirect

Add missing "fr" and "fr-FR" locales in languages.yml

wiz committed:
Replace absolute URL references with relative URL references

Commit:
Merge pull request #232 from wiz/use-relative-urls

Replace absolute URL references with relative URL references

wiz committed:
Localize site name/description for metadata preview pane usage

erciccione commented on _data/fa.yml:
```diff
@@ -0,0 +1,361 @@
+  hBisqDAO: The Bisq DAO
+  hBisqDaoWhat: "BISQ DAO چیست؟"
+  pBisqDaoWhat: >
+    <p>BISQ DAO (سازمان خودمختار غیر متمرکز) یک مکانیزم اداری غیرمتمرکز برای نرم افزار Bisq است. که برای یک شرکت است. اگر یک نرم افزار غیر متمرکز توسط یک نهاد واحد مثل یک شرکت کنترل شود, چقدر می تواند مفید باشد؟</p> <p>یک نرم افزار غیر متمرکز بدون بدون حکمرانی غیر متمرکز مفید واقع نخواهد بود.</p> <p>به همین دلیل است که Bisq یک شرکت یا نهاد قانونی از هر نوعی نیست - بلکه به عنوان یک DAO سازماندهی ش ...
```
You open a hyperlink at the end of this string, but you don't close it. This is messing up the format of the section that follows. I removed the link locally while testing and everything worked fine.

huey735 replied:
Thanks, @erciccione. After some searching I've found it yesterday. @Blockchainsmoker has taken care of it and I'll update it soon.

erciccione reviewed (changes requested):
Found the problem. See comment

Commit:
Fix header on 404 page

Commit:
Merge pull request #230 from wiz/localized-screenshots

Display localized app screenshots

Commit:
Merge pull request #233 from wiz/localized-metadata-for-preview

Localize site name/description for metadata preview pane usage

Commit:
Merge pull request #234 from m52go/fix-404

Fix header on 404 page

wiz committed:
Avoid relative image URL confusion by referencing /images directly

wiz committed:
Undo locale specific image renaming workaround

Commit:
Merge pull request #235 from wiz/fix-relative-image-urls

Avoid relative image URL confusion by referencing /images directly

Commit:
Merge pull request #229 from wiz/japanese

Add Japanese localizations

huey735 committed:
Rename dao images and fix text

huey735 committed:
Rename dao images and fix text

huey735 committed:
Translate site name/description for metadata preview pane usage

Commit:
Merge pull request #236 from huey735/addGermanMetadata

Translate site name/description for metadata preview pane usage

huey735 committed:
Edit Outdated_Translation tool text

Commit:
Merge pull request #237 from huey735/editOutdatedTranslationTool

Edit Outdated_Translation tool text

Commit:
Correct altcoin volume

wiz committed:
Add keybase team link in community footer

wiz committed:
Replace broken relative screenshot URL with absolute URL

Commit:
Merge pull request #240 from wiz/fix-localized-screenshots-url

Replace broken relative screenshot URL with absolute URL

wiz committed:
Add Keybase icon link in footer templates as well

Commit:
Add placeholder images for french front page

wiz committed:
Display name of languages and country flags in locale selector

- Use full language name instead of two letter language code
- Use flag emoji instead of two letter country code
- Fix sort order of locale selector pull-down

Commit:
Merge pull request #239 from wiz/add-keybase-link

Add keybase team link in community footer

Commit:
Merge pull request #242 from wiz/fix-language-selector

Display name of languages and country flag in locale selector

Commit:
Merge pull request #238 from m52go/correct-altcoin-volume

Correct altcoin volume

Commit:
Merge pull request #241 from m52go/add-french-placeholders

Add placeholder images for french front page

Commit:
Fix fee schedule charts on dao blog post

BTC fee numbers were switched, and there was a duplicate column for BTC
taker fees.

Commit:
Merge pull request #245 from m52go/fix-blog-dao-fees

Fix fee schedule charts on dao blog post

ripcurlx committed:
Updates for v1.1.6 release

ripcurlx committed:
Merge pull request #247 from ripcurlx/updates-for-v1.1.6

Updates for v1.1.6 release

huey735 committed:
Fix typos and add variables

I fixed some typos on the text and added the new front matter
variables that had been added since starting this PR:
flag, outdated_translation.

huey735 committed:
Merge branch 'addFarsi' of github.com:huey735/bisq-website into addFarsi
